nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/AbstractHadoopProcessor.java

```diff
@@ -191,9 +191,19 @@ public final void abstractOnScheduled(ProcessContext context) throws IOException
             HdfsResources resources = hdfsResources.get();
             if (resources.getConfiguration() == null) {
                 String configResources = context.getProperty(HADOOP_CONFIGURATION_RESOURCES).getValue();
-                String dir = context.getProperty(DIRECTORY_PROP_NAME).getValue();
-                dir = dir == null ? "/" : dir;
-                resources = resetHDFSResources(configResources, dir, context);
+                final String dir;
+                final PropertyDescriptor directoryPropDescriptor = context.getProperties().keySet().stream()
```

Comment:
@jtstorck I think this can be made a lot simpler by simply calling getPropertyDescriptor(DIRECTORY_PROP_NAME) of the AbstractProcessor super class, no?